tests/VolleyManagement.Specs/TeamsContext/EditTeamSteps.cs

```diff
@@ -0,0 +1,149 @@
+                Name = "TestName",
+                Coach = "New coach",
+                Achievements = "Achive",
+                Captain = _player,
+                Players = new List<PlayerEntity>()
+            };
+        }
+
+        [Given(@"(.*) team exists")]
+        public void GivenTeamExists(string name)
+        {
+            _team.Name = name;
+            TestDbAdapter.CreateTeamWithCaptain(_team, "First", "Last");
+            _player = _team.Captain;
+            _team.Players.Add(_player);
+        }
+
+        [Given(@"name set to Very Looong team name which should be more than (.*) symbols")]
+        public void GivenNameChangedToNameWhichShouldBeMoreThan(int newNameLength)
+        {
+            var newName = new string('n', newNameLength + 1);
+            _team.Name = newName;
+        }
+
+        [Given(@"Team (.*) team does not exist")]
+        public void GivenTeamDoesNotExist(string name)
+        {
+            _team.Name = name;
+            _team.Captain = _player;
+        }
+
+        [Given(@"name changed to (.*)")]
+        public void GivenNameChangedToAnother(string newName)
+        {
+            _team.Name = newName;
+        }
+
+        [Given(@"captain is changed to (.*)")]
+        public void GivenCaptainIsChangedToAnother(string captainName)
+        {
+            var names = SpecsHelper.SplitFullNameToFirstLastNames(captainName);
+
+            _captain = new PlayerEntity {
+                FirstName = names.FirstName,
+                LastName = names.LastName,
+            };
+
+            TestDbAdapter.CreatePlayer(_captain);
+            _team.Players.Add(_captain);
+        }
+
+        [When(@"I execute EditTeam")]
+        public void WhenIExecuteEditTeam()
+        {
+            try
+            {
+                var team = new Team(_team.Id,
+                    _team.Name,
+                    _team.Coach,
+                    _team.Achievements,
+                    new PlayerId(_team.Captain.Id),
+                    new List<PlayerId>()
+                );
+
+                _teamService.Edit(team);
+            }
+            catch (Exception exception)
+            {
+                _exception = exception;
+                _isExceptionThrown = true;
+            }
+        }
+
+        [When(@"I execute ChangeTeamCaptain")]
+        public void WhenIExecuteChangeTeamCaptain()
+        {
+            _teamService.ChangeCaptain(new TeamId(_team.Id), new PlayerId(_captain.Id));
+            _team.Captain = _captain;
+            _team.CaptainId = _captain.Id;
+        }
+
+        [Then(@"team is updated succesfully")]
+        public void ThenTeamIsUpdatedSuccesfully()
+        {
+            using (var context = TestDbAdapter.Context)
+            {
+
+                var teamEntity = context.Teams.Find(_team.Id);      
+                teamEntity.Should().BeEquivalentTo(_team,options=>options.Including(x=>x.CaptainId).Including(x=>x.Name));
```

Comment:
why options include only captain and name?

Reply:
Because we need check only theese two case